build/Nuspec/NuGet.Localization.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
put this file under src as well. Create a folder for the project name and put it under there.